tsgen/__main__.py

```diff
@@ -66,11 +71,11 @@ def generate_c_code(self):
         Return:
             Tuple containing paths to C header and source files
         """
-        HEADER_FILE = f'{self.C_DATABASE_NAME}.h'
-        SOURCE_FILE = f'{self.C_DATABASE_NAME}.c'
+        HEADER_FILE = f'{self.C_DATABASE_NAME + bus}.h'
```

Comment:
See prev comment about database names, this is the code that will need to be tweaked.